lib/src/variant_utils.rs

```diff
@@ -32,36 +32,21 @@ pub fn variant_normal_from_bytes(ty: &str, bytes: glib::Bytes) -> glib::Variant
 
 /// Create a new `ay` GVariant.
 pub fn new_variant_bytearray(buf: &[u8]) -> glib::Variant {
-    unsafe {
-        let r = glib_sys::g_variant_new_fixed_array(
-            b"y\0".as_ptr() as *const _,
-            buf.as_ptr() as *const _,
-            buf.len(),
-            size_of::<u8>(),
-        );
-        glib_sys::g_variant_ref_sink(r);
-        from_glib_full(r)
-    }
+    buf.to_variant()
```

Comment:
Being able to use stuff like this was a primary motivation for the bump - look ma no `unsafe`!

Reply:
And done!  Almost all of `variant_utils.rs` is gone, and the only `Variant` bits are just `pub(crate)`.  There's still `VariantDictExt` which I think is kind of still useful...needs some investigation.